Tella/Data/Database/Common/DatabaseConstants.swift

```diff
@@ -65,6 +70,18 @@ struct D {
     static let cCreatedDate = "c_created_date"
     static let cUpatedDate = "c_upated_date"
     static let cUpdatedDate = "c_updated_date"
+    // MARK: Uwazi Locale
+    static let cLocale = "c_locale"
+    static let cLocaleId = "c_locale_id"
+
+    static let cTemplateId = "c_template_id"
+    static let cEntity = "c_entity"
+    static let cDownloaded = "c_downloaded"
+    static let cUpdated = "c_updated"
+    static let cFavorite = "c_favorite"
+    static let cServerName = "c_server_name"
+    static let cCookie = "c_cookie"
```

Comment:
I think it's better to save the connect cookies `C_CONNECT_COOKIES` and locale cookies `C_LOCALE_COOKIES` separately

Reply:
yeah we are actually saving it separated: 

```
let columns = [
            cddl(D.cServerId, D.integer, primaryKey: true, autoIncrement: true),
            cddl(D.cName, D.text),
            cddl(D.cURL, D.text),
            cddl(D.cUsername, D.text),
            cddl(D.cPassword, D.text),
            cddl(D.cAccessToken, D.text),
            cddl(D.cLocale, D.text),
            cddl(D.cServerType, D.integer)
        ]
   ```
   Then I'm generating dynamically the cookie with this function:
   
   private func createCookie() -> String {
        let accessTokenValue = accessToken ?? ""
        let localeValue = locale ?? ""
        return "connect.sid=\(accessTokenValue);locale=\(localeValue)"
    }
    
    I think I forgot to remove the constant from a previous implementation, thanks for catching the typo!